src/sdk-reference/go/1/getting-started/snippets/document.go

```diff
@@ -0,0 +1,51 @@
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"log"
+	"os"
+
+	"github.com/kuzzleio/sdk-go/kuzzle"
+	"github.com/kuzzleio/sdk-go/protocol/websocket"
+)
+
+func main() {
+	// Create websocket protocol, replace "kuzzle" with
+	// your Kuzzle hostname like "localhost"
+	c := websocket.NewWebSocket("kuzzle", nil)
```

Comment:
Cannot we use "localhost" for all snippets (and remove the comment above)?

Reply:
In the test framework, the docker container know Kuzzle container as `kuzzle`, if you have a workaround to have `localhost` instead I'm in

Reply:
Ah.. true, sorry.